README.md

```diff
@@ -121,6 +121,8 @@ The devices below are fully feature supported by OpenRazer, which means all avai
 | Razer Blade 18 (2024)                         |  1532:02B8  |
 | Razer Blade 14 (2025)                         |  1532:02C5  |
 | Razer Blade 16 (2025)                         |  1532:02C6  |
+| Razer BlackWidow V4 Tenkeyless (wireless)     |  1532:02D5  |
+| Razer BlackWidow V4 Tenkeyless (wired)        |  1532:02D7  |
```

Comment:
Is the device name not `Razer BlackWidow V4 Tenkeyless HyperSpeed`? Also please uppercase Wireless/Wired to make the labels consistent